ethpm_types/contract_type.py

```diff
@@ -187,14 +260,31 @@ class ContractType(BaseModel):
     then ``MyContract`` would be the type.
     """
 
-    # NOTE: Field is optional if `ContractAlias` is the same as `ContractName`
     name: Optional[str] = Field(None, alias="contractName")
+    """
+    The name of the contract type. The field is optional if ``ContractAlias``
+    is the same as ``ContractName``.
+    """
+
     source_id: Optional[str] = Field(None, alias="sourceId")
+    """
+    The global source identifier for the source file from which this contract type was generated.
+    """
+
     deployment_bytecode: Optional[Bytecode] = Field(None, alias="deploymentBytecode")
+    """The bytecode for the ContractType."""
+
     runtime_bytecode: Optional[Bytecode] = Field(None, alias="runtimeBytecode")
-    # abi, userdoc and devdoc must conform to spec
+    """The unlinked 0x-prefixed runtime portion of bytecode for this ContractType."""
+
     abi: List[ABI] = []
+    """The application binary interface to the contract."""
+
     sourcemap: Optional[SourceMap] = None  # NOTE: Not a part of canonical EIP-2678 spec
+    """
+    **NOTE**: This is not part of the canonical EIP-2678 spec
```

Comment:
This is noted above, I would say the docstrings here should have some substance, and either keep the comment note or move it into the docstring